### DISCUSSION TITLE
Ability to hot reload front end

### DISCUSSION THREAD

**Discussion by Rafii2198** [tauri-apps/tauri#5383] [Q&A] [answered]
I believe when you use built tools (Vite for example), it's possible to hot reload your frontend whenever you make a change. Is there a way to achieve this while using just plain HTML? I have seen some tutorials using Node.js, but I don't use it.

**Comment by amrbashir** [comment, marked as answer]
This is now implemented and will be available in the next release of tauri cli.

**Reply by Rafii2198** [reply]
Oh wow, I didn't expect that! Do we have ETA of the next release?

**Reply by amrbashir** [reply]
unfortunately not
